go.mod

```diff
@@ -204,3 +204,5 @@ replace google.golang.org/grpc v1.54.0 => google.golang.org/grpc v1.26.0
 // kvproto at the same time. You can run `go mod tidy` to make it replaced with go-mod style specification.
 // After the PR to kvproto is merged, remember to comment this out and run `go mod tidy`.
 // replace github.com/pingcap/kvproto => github.com/$YourPrivateRepo $YourPrivateBranch
+
+replace github.com/pingcap/kvproto => github.com/rleungx/kvproto v0.0.0-20230516051126-0db8e6f4780d
```

Comment:
https://github.com/pingcap/kvproto/pull/1116 is merged

Reply:
upgraded to kvproto/master.